lmbdtest/test_downsample_volume_lambda.py

```diff
@@ -15,19 +15,55 @@
 # lambdafcns contains symbolic links to lambda functions in boss-tools/lambda.
 # Since lambda is a reserved word, this allows importing from that folder 
 # without updating scripts responsible for deploying the lambda code.
-from lambdafcns.downsample_volume import (
-    S3DynamoDBTable, downsample_volume, S3Bucket
-)
 
-from bossutils.multidimensional import XYZ
-import numpy as np
+import importlib
+# from lambda import downsample_volume as dsv
+dsv = importlib.import_module("lambda.downsample_volume")
+
 import unittest
 from unittest.mock import patch
 
+from bossutils.multidimensional import XYZ
+import numpy as np
+import blosc
+
+import boto3
+from moto import mock_s3, mock_sqs, mock_dynamodb2
+
+# Downsample Volume Lambda handler arguments
+args = {
+    'bucket_size': 1,
+    'cubes_arn': 'cubes_arn',
+
+    'args': {
+        'collection_id': 1,
+        'experiment_id': 1,
+        'channel_id': 1,
+        'annotation_channel': False,
+        'data_type': 'uint8',
+
+        's3_bucket': 's3_bucket',
+        's3_index': 's3_index',
+
+        'resolution': 0,
+
+        'type': 'anisotropic',
```

Comment:
Should there be a test for isotropic data? Does partial downsample work for both types?